### ISSUE TITLE
Role Scoper and CCTM conflict.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [viktorthang/wordpress-custom-content-type-manager#429]
```
Hi,
I"M SEEING THESE ERRORS WHEN role Scoper and CCTM are activated.:

Warning: Illegal string offset 'read_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 147

Warning: Illegal string offset 'edit_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 148

Warning: Illegal string offset 'delete_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 149

Warning: Illegal string offset 'read_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 150

Warning: Illegal string offset 'edit_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 151

Warning: Illegal string offset 'delete_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 152

Warning: Illegal string offset 'read_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 147

Warning: Illegal string offset 'edit_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 148

Warning: Illegal string offset 'delete_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 149

Warning: Illegal string offset 'read_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 150

Warning: Illegal string offset 'edit_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 151

Warning: Illegal string offset 'delete_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 152

Warning: Illegal string offset 'read_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 147

Warning: Illegal string offset 'edit_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 148

Warning: Illegal string offset 'delete_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 149

Warning: Illegal string offset 'read_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 150

Warning: Illegal string offset 'edit_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 151

Warning: Illegal string offset 'delete_post' in 
C:\xampp\htdocs\bp-scene\wp-content\plugins\role-scoper\wp-cap-helper_cr.php on 
line 152

FOR THIS CODE:

// This simplifies map_meta_cap handling. Actual attachment or revision editing 
access is still based on access to the parent post
if ( ! defined( 'SCOPER_RETAIN_CUSTOM_METACAPS' ) ) {
Line 147:    $type_caps['attachment']['read_post'] = 'read_attachment';
$type_caps['attachment']['edit_post'] = 'edit_attachment';
$type_caps['attachment']['delete_post'] = 'delete_attachment';
$type_caps['revision']['read_post'] = 'read_revision';
$type_caps['revision']['edit_post'] = 'edit_revision';
$type_caps['revision']['delete_post'] = 'delete_revision';
}
Word Press 3.4.2; Buddypress1.6.1; Theme, Custom Community 1.10; CCTM 0.9.6.1; 
Role Scoper 1.3.5.7; PHP 5.4.4.

I have one custom content type configured.
Any ideas?

Regards, Jobey

```

Original issue reported on code.google.com by `DrewSlat...@gmail.com` on 20 Nov 2012 at 8:09

**Comment by GoogleCodeExporter** [comment]
```
All those errors are in the other plugin, so there's probably not much I can 
do: they aren't correctly checking whether or not an array key exists.

This will probably be resolved with issue #409, but again, the error is not in 
my code.
```

Original comment by `ever...@fireproofsocks.com` on 20 Nov 2012 at 10:07
- Changed state: **InReview**

**Comment by GoogleCodeExporter** [comment]
```
I'm going to mark this as closed -- I think the flexibility to coexist is there 
now that issue #409 is closed (as of version 0.9.6.5).
```

Original comment by `ever...@fireproofsocks.com` on 4 Dec 2012 at 11:22
- Changed state: **Fixed**
- Added labels: **Milestone-Release0.9.7**
